### ISSUE TITLE
Schaltfläche für Änderungsantrag wird bei bestimmter Positionierung der Antragsabschnitte falsch angezeigt

### ISSUE THREAD

**Issue by FloJuh** [CatoTH/antragsgruen#986]
Wenn bei zwei Antragsabschnitten die Positionen im PDF wie folgt ausgewählt werden:
- Ein Antragsabschnitt wird auf „Links“ positioniert,
- Ein anderer Antragsabschnitt wird auf „Rechts“ positioniert, 

ist die Schaltfläche für die Anzeige des Änderungsantrags über dem Antragsabschnitt auf der „Rechts“-Seite platziert, was zu einer falschen Darstellung führt. Diese Anordnung beeinträchtigt die Benutzerfreundlichkeit und kann zu Verwirrung führen.

### Screenshots

1. Fehlerhafte Darstellung:

![SharedScreenshot](https://github.com/user-attachments/assets/97eb8264-af06-4fbf-b577-a79cc641b0db)

2. Backend-Einstellung:

![SharedScreenshot2](https://github.com/user-attachments/assets/cf95db85-cdb0-4e86-8260-32344a2e0aa0)

3. Normale Darstellung ohne Positionierung der Antragsabschnitte:

![Screenshot (21)](https://github.com/user-attachments/assets/2f8cf7b1-d5eb-40de-85d3-dd853d779dfb)

**Comment by CatoTH** [comment]
Ja, ist nicht elegant. Ich frage mich gerade nur, ist das ein realistischer Anwendungsfall? Kommt das jetzt vom "herumspielen" mit den Einstellungen, oder wollt ihr das wirklich so einsetzen?
Das zweispaltige kommt in der Praxis ja eher bei Bewerbungen zum Einsatz, und die sind meistens nicht änderbar...

**Comment by FloJuh** [comment]
Wir nutzen das für Satzungsänderungsanträge. So wie auf den Bildern zu sehen: Antragstext/neue Satzung links und aktuelle Satzung rechts. 
Unsere Mitglieder fanden die Darstellung der Satzungsänderungsfunktion verwirrend.
